server/src/test/java/com/google/fhir/gateway/BearerAuthorizationInterceptorTest.java

```diff
@@ -138,6 +145,17 @@ public void authorizeRequestPatient() throws IOException {
     assertThat(testPatientJson, equalTo(writerStub.toString()));
   }
 
+  @Test
+  public void authorizeRequestPatientAuditEventLoggingEnabledNoUserWho() throws IOException {
```

Comment:
I think this is not a very useful test because it seems it doesn't exercise the `getUserWho != null` as shown in [this coverage report](https://app.codecov.io/gh/google/fhir-gateway/pull/406/blob/server/src/main/java/com/google/fhir/gateway/BearerAuthorizationInterceptor.java) (maybe because there is a `JWTDecodeException`). Is my understanding correct and if yes can you please fix this such that this test covers most of the new code in `BearerAuthorizationInterceptor`?

Reply:
Done